Jenkinsfile

```
@@ -1,7 +1,7 @@
 #!/usr/bin/groovy
-/* groovylint-disable DuplicateMapLiteral, DuplicateNumberLiteral
-   groovylint-disable DuplicateStringLiteral, NestedBlockDepth, VariableName
-*/
+/* groovylint-disable-next-line LineLength */
```

I'm not getting LineLength warnings with the default plugin but this looks like it disables it for one line only, do you mean this?
```suggestion
/* groovylint-disable LineLength */
```

---

Yes, it was intentional and now superfluous and benign with the (previously) long groovylint comment broken into multiple lines.  Somebody previously broke the line up into multiple lines and didn't remove that suppression.  It's not really causing any harm though.